apps/aecore/src/aec_db.erl

```diff
@@ -761,10 +761,8 @@ assert_schema_node_name(#{persist := true}) ->
     case DbOwnerNode =:= node() of
         true -> ok;
         false ->
-            DbDir = case aeu_env:user_config([<<"chain">>, <<"db_path">>]) of
-                        undefined -> "data";
-                        {ok, Path} -> Path
-                    end,
+            {ok, DbDir} = aeu_env:find_config([<<"chain">>, <<"db_path">>],
+                                              [user_config, schema_default]),
```

Comment:
Acutually what is used in the code is https://github.com/aeternity/aeternity/blob/master/apps/aecore/src/aecore_app.erl#L62, it does take into account `schema_default`.